jwql/utils/constants.py

```diff
@@ -42,8 +42,19 @@
 JWST_MAST_SERVICES = ['Mast.Jwst.Filtered.{}'.format(value.title()) for value in
                       JWST_INSTRUMENT_NAMES]
 
-FILE_SUFFIX_TYPES = ['uncal', 'cal', 'rateints', 'rate', 'trapsfilled',
-                     'i2d', 'x1d', 's2d', 's3d', 'dark', 'ami', 'crf']
+# Add guider suffixes (see JWST-STScI-004800, SM-12)
+FILE_SUFFIX_TYPES = ['stream', 'stacked_uncal', 'image_uncal', 'stacked_cal', 'image_cal']
```

Comment:
Since you are separating into different suffix types anyways, I suggest to store these in separate variables, e.g. `GUIDER_SUFFIX_TYPES =  ['stream', 'stacked_uncal', 'image_uncal', 'stacked_cal', 'image_cal']` etc. and then construct FILE_SUFFIX_TYPES at the end
`FILE_SUFFIX_TYPES = GUIDER_SUFFIX_TYPES + ...`

I cannot yet thing about a use case, but it may even be useful to have the functionality of determining the suffix group based on a given type (i.e. I have a file suffix `amiavg` and have jwql tell me what this is), so one could think about a dictionary to store these lists.

Reply:
Good idea - done!